gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -174,7 +176,7 @@ def self.sig_validation_error_handler=(value)
   end
 
   def self.sig_validation_error_handler(error, opts={})
-    if @sig_validation_error_handler
+    if defined?(@sig_validation_error_handler) && @sig_validation_error_handler
```

Comment:
ooc: why is this `defined?` needed

Reply:
That's a mistake, I should have pre-initialized it at the class level like the others.

Reply:
I fixed it.